### DISCUSSION TITLE
Sort for last Commented

### DISCUSSION THREAD

**Discussion by myzinsky** [directus/directus#3918] [Feature Requests]
It would be a nice to have a feature to sort content entries according to a last commented filter.

**Comment by benhaynes** [comment]
Interesting idea, though that metadata is not relationally joined to the items... so it's a bit more complex than that.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
